csharp/HOLMS.Platform/HOLMS.Platform/Support/Security/ValidatedJWToken.cs

```diff
@@ -57,7 +57,7 @@ public class ValidatedJWToken {
 
             var expirationClaim = tok.Claims.FirstOrDefault(x => x.Type == JwtRegisteredClaimNames.Exp);
             if (expirationClaim != null) {
-                if (tok.ValidTo < DateTime.Now) {
+                if (tok.ValidTo < DateTime.UtcNow) {
                     throw new SecurityException("Presented token is expired");
```

Comment:
Can we refine this class hierarchy a bit, and make `TokenExpiredException` subclass `SecurityException`?

I think this is sufficiently "baked" to warrant a little more detail work on these exceptions.